OnShift/SendBird-JavaScript
# PL 5 unit test framework

# Things Done
- added jest test runner (per the recommendations of the Outback team) to the widget
- added minimal tests around one function in util file (smallest I could think to add)
- added coverage report that we can run

# How To Test
- `npm test`
- `npm run coverage`
- make sure the widget still behaves correctly
